libs/chiliui/chili/controls/editbox.lua

```diff
@@ -81,21 +91,25 @@ local inherited = this.inherited
 
 function EditBox:New(obj)
 	obj = inherited.New(self, obj)
-	obj._interactedTime = Spring.GetTimer()
+	obj._interactedTime = spGetTimer()
 		--// create font
-	obj.hintFont = Font:New(obj.hintFont)
-	obj.hintFont:SetParent(obj)
+	if obj.hint then
+		obj.hintFont = Font:New(obj.hintFont)
+		obj.hintFont:SetParent(obj)
+	else
+	  obj.hintFont = false
```

Comment:
Avoiding "getting" the hint font when you have no hint was helpful.

Reply:
What if the hint is later set?